collections/card4l.yaml

```diff
@@ -0,0 +1,122 @@
+ The twin Sentinel-1A and 1B satellites are designed to acquire data with a revisit cycle of 6 to 12 days providing continuous all-weather, day and night imagery for Land and Maritime Monitoring. Archived Sentinel-1 CARD4L NRB data VV and VH polarization over Africa are provided, with new acquisitions being updated daily as they become available. To obtain Sentinel-1 CARD4L NRB data for other parts of the world, registered users have the possibility of custom processing using [CARD4L generation tool](https://apps.sentinel-hub.com/s1-card4l/). 
+AdditionalInfoExternal:
+    Title: Additional info 
+    Path: card4l/README.MD 
+Image: card4l/thumbnail.png        
+Resolution: 20m
+GeographicalCoverage: Africa Continent
+TemporalResolution: Every 6 to 12 days
+TemporalAvailability: July 01, 2018 - Ongoing
+UpdateFrequency: Daily
+BandInformation: 
+  Table:
+      Columns:
+        - Name: Name
+          Title: Name
+        - Name: Description
+          Title: Description        
+      Rows:
+        - Name: VV
+          Description: Linear backscatter intensity in VV polarization.
+        - Name: VH
+          Description: Linear backscatter intensity in VH polarization.
+        - Name: MASK
+          Description: Data mask.
+        - Name: AREA
+          Description: Normalized scattering area.
+        - Name: ANGLE
+          Description: Local incidence angle (degree).
+Contact: https://forum.sentinel-hub.com/c/euro-data-cube/
+Provider: "[Digital Earth Africa](https://www.digitalearthafrica.org/)"
+ManagedBy: "[Sentinel Hub](https://www.sentinel-hub.com/)"
+Tags:
+  - sentinel hub
+  - xcube
+  - raster
+  - systematic
+  - satellite imagery
+  - SAR
+  - radar
+  - backscattering
+  - polarization
+  - Copernicus
+  - maritime monitoring
+  - land monitoring
+  - disaster response
+  - open data
+  - sentinel
+  - card4l
+  - radiometric terrain correction
+License: Access to Sentinel data is free, full and open for the broad Regional, National, European and International user community. Read [Terms and Conditions](https://scihub.copernicus.eu/twiki/do/view/SciHubWebPortal/TermsConditions).
+LicenseType: proprietary  
+LicenseUrl: https://scihub.copernicus.eu/twiki/do/view/SciHubWebPortal/TermsConditions
+Resources:
+  - Group: Sentinel Hub Resources
+    EndPoint: https://creodias.sentinel-hub.com
```

Comment:
Thanks for updating this!

I changed that for other collections in https://github.com/eurodatacube/public-collections/pull/168 but unfortunately I forgot to mention this change in the `CONTRIBUTION.md` which I will do shortly.

This can be resolved.